lib/ui/login/login_screen.dart

```diff
@@ -173,15 +156,27 @@ class LoginScreenState extends ConsumerState<LoginScreen>
                   right: AppDimensions.spacing24,
                   bottom: AppDimensions.spacing20,
                 ),
-                child: _configuredTextField(context, true),
+                child: customTextField(
+                  context,
+                  _emailTextFieldController,
+                  TextInputType.emailAddress,
+                  false,
+                  AppLocalizations.of(context)!.email,
```

Comment:
(Optional) I think it will be a bit easier to read
```suggestion
                  context: context,
                  controller: _emailTextFieldController,
                  textInputType: TextInputType.emailAddress,
                  isObscuredText: false,
                  hintText: AppLocalizations.of(context)!.email,
```